edb/server/server.py

```diff
@@ -552,6 +553,19 @@ def _idle_gc_collector(self):
             raise
 
     async def _create_compiler_pool(self):
+        # Force Postgres version in BackendRuntimeParams to be the
+        # minimal supported, because the compiler does not rely on
+        # the version, and not pinning it would make the remote compiler
+        # pool refuse connections from clients that have differing versions
+        # of Postgres backing them.
+        runtime_params = self.get_backend_runtime_params()
+        min_ver = '.'.join(str(v) for v in defines.MIN_POSTGRES_VERSION)
+        runtime_params = runtime_params._replace(
```

Comment:
Shouldn't we put this after into the `args` dict below?

Reply:
:facepalm: